demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1451,6 +1451,9 @@ def has_duplicate(id_set_subset_list, id_to_check, object_type=None, print_logs=
             if dict1.get('kind', '') != dict2.get('kind', ''):
                 return False
 
+        if dict1.get('pack') == dict2.get('pack'):
```

Comment:
didn't we say in this case it shouldn't get to the `united_id_set`? but in this case it does.
Also please add a dev comment here explaining this.

Reply:
It is not considered as duplicates, and it is not added to united_id_set - 
Because I added a test that makes sure there is no such entry already on the list.
The reason it should not be added to the united_id_set is that such an entity already exists there.

Reply:
I think you better change the condition to be 
```
if dict1.get('pack') == dict2.get('pack'):
    return True
```
This way the for will run over all the duplication combinations and if there are two non identical packs, it will return True